lnwire/neighbor_rst_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package lnwire
+
+import (
+       "bytes"
+       "testing"
+
+       "github.com/roasbeef/btcd/wire"
+)
+
+func TestNeighborRstMessageEncodeDecode(t *testing.T) {
+       b := new(bytes.Buffer)
+       msg1 := NeighborRstMessage{}
+       err := msg1.Encode(b, 0)
+       if err != nil {
+               t.Fatalf("Can't encode message ", err)
+       }
+       msg2 := new(NeighborRstMessage)
+       err = msg2.Decode(b, 0)
+       if err != nil {
+               t.Fatalf("Can't decode message %v", err)
+       }
```

Comment:
There is no information inside NeighborRST message. So all messages of this type are equal, so there is no need to compare.